src/apps/core/templates/core/account-verified.html

```diff
@@ -1,16 +1,35 @@
 {% extends 'base.html' %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
 
 {% load static %}
 
 {% block style %}
 <link rel="stylesheet" href="{% static 'core/account-verified/style.css' %}">
 {% endblock style %}
 
-{% block content %}
+</head>
+<body>
```

Comment:
Remove this code as well
```suggestion
</head>
<body>
```